### PR TITLE
add: e2e tests

### PR DESCRIPTION
**Pull request by rajarju**
## Description

Adding e2e tests for lib
- bundle sample web app
- deploy 
- undeploy 

## Related Issue
Fix: https://github.com/adobe/aio-lib-web/issues/144

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-standard": "^4.0.1",
     "jest": "^24.9.0",
     "jsdoc-to-markdown": "^6.0.1",
-    "memfs": "^3.0.1"
+    "memfs": "^3.0.1",
+    "superagent": "^6.1.0"
```
Comment (moritzraho) [inline package.json:57]: this should be a dev dependency

File: e2e/sample-app/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('Hello')
```
Comment (moritzraho) [inline e2e/sample-app/index.js:1]: why do you need an index.js at the root of the app ?

**Review by moritzraho — COMMENTED** [review]
few comments, otherwise lgtm !

File: e2e/sample-app/index.js (hunk 2 above)
Comment (rajarju) [reply e2e/sample-app/index.js:1]: @moritzraho 
No specific reason, Just added it to check the build output.

File: package.json (hunk 1 above)
Comment (rajarju) [reply package.json:57]: @moritzraho It is part of devDependencies

File: e2e/sample-app/index.js (hunk 2 above)
Comment (moritzraho) [reply e2e/sample-app/index.js:1]: okie maybe let's remove it then  if the tests pass now :)

Comment (rajarju) [reply e2e/sample-app/index.js:1]: The test will fail, 
It looks for the build output for this file

**Comment by rajarju** [comment]
@moritzraho Updated: Replaced superagent with node-fetch

**Review by moritzraho — APPROVED** [review]

**Pull request merged**